crates/sui-core/src/authority_client.rs

```diff
@@ -412,9 +413,9 @@ impl AuthorityAPI for LocalAuthorityClient {
     ) -> Result<TransactionInfoResponse, SuiError> {
         let state = self.state.clone();
         let fault_config = self.fault_config;
-        tokio::spawn(
-            async move { Self::handle_certificate(state, certificate, fault_config).await },
-        )
+        spawn_monitored_task!(async move {
+            Self::handle_certificate(state, certificate, fault_config).await
```

Comment:
Nit: here (and in many similar places) we don't really need `async move` block.
For example this line is equivalent: 

```
spawn_monitored_task!(Self::handle_certificate(state, certificate, fault_config))
```

I know this seem to be a pattern across all the code in sui, but since you are touching those places perhaps you can fix it as well?

Reply:
fixed a bunch of these cases.